Resources/config/plugins.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="httplug.plugin.cache" class="Http\Client\Plugin\CachePlugin" public="false">
```

Comment:
agree that the plugins should be private (= removed from the container after it is built). we should list the services that are available in the documentation, and also remind that they are private and can thus not be manually fetched from the container.